modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc

```diff
@@ -0,0 +1,95 @@
+[id="proc-add-custom-app-config-file-ocp_{context}"]
+= Adding a custom application configuration file to {ocp-short} using the Operator
+
+A custom application configuration file is a ConfigMap that you can use to change the configuration of your {product} instance to meet the needs and preferences of your organization. If you are deploying your {product-short} instance on {ocp-brand-name}, you can use the {product} Operator to add a custom application configuration file to your {ocp-short} instance by creating the ConfigMap and referencing it in the {product-short} Custom Resource.
+
+The custom application configuration file contains a sensitive environment variable, named `BACKEND_SECRET`, which is a mandatory backend authentication key that {product-short} uses to reference an environment variable defined in an {ocp-short} Secret. You must create a custom Secret, named 'secrets-rhdh', and reference it in the {product-short} Custom Resource.
```

Comment:
```suggestion
The custom application configuration file contains a sensitive environment variable named `BACKEND_SECRET`. This variable contains a mandatory backend authentication key that {product-short} uses to reference an environment variable defined in an {ocp-short} secret. You must create a secret named 'secrets-rhdh', and reference it in the {product-short} CR.
```

Suggestion to break up sentences so that this is easier to parse.
Similar to above comment, unless referring to a specific `Secret` object, this should be lowercase "secret" for OCP references.